---
lidar_subscriber_topic: /lidar
navigation_subscriber_topic: /navigation
dock_status_subscriber_topic: /dock_state
bumper_status_subscriber_topic: /bumper_status
publisher_topic: /beliefs
update_rate: 0.1
queue_size: 10